Source/ARTRealtime+Private.h

```diff
@@ -95,7 +95,7 @@ NS_ASSUME_NONNULL_BEGIN
 @property (readonly, getter=getClientOptions) ARTClientOptions *options;
 
 /// Suspend the behavior defined in RTN15a, that is trying to immediately reconnect after a disconnection
-@property (readwrite, assign, nonatomic) BOOL suspendImmediateReconnection;
+@property (readwrite, assign, nonatomic) BOOL disableImmediateReconnection;
```

Comment:
Extreme nitpick: a verb makes me think of a method; a noun makes me think of a variable. So, `immediateReconnectionDisabled` (or `Enabled`).

Reply:
I would prefer `Enabled` to `Disabled` in the name, as I think it makes it easier to understand the use.

Or, perhaps, use a prefix like `shouldImmediatelyReconnect`

Reply:
I like that more: `shouldImmediatelyReconnect`. Thanks folks.

Reply:
Done 69e5cf0.